pkg/ibmvpc/parser.go

```diff
@@ -651,45 +684,46 @@ func getSGconfig(rc *ResourcesContainer,
 }
 
 func getNACLconfig(rc *ResourcesContainer,
-	res *vpcmodel.CloudConfig,
+	res map[string]*vpcmodel.CloudConfig,
 	subnetNameToSubnet map[string]*Subnet,
 	skipByVPC func(string) bool) error {
 	// nacl
-	naclList := []*NACL{}
-	var layerVPC *VPC
-	for i := range rc.naclList {
-		nacl := rc.naclList[i]
+	naclList := map[string][]*NACL{} // map from vpc name to its nacls
+	for vpcnName := range res {
+		naclList[vpcnName] = []*NACL{}
+	}
```

Comment:
Is this required? Appending to nil should work as well.

Reply:
right, done.